.github/workflows/consecutive.yml

```diff
@@ -23,7 +23,7 @@ on:
         type: choice
         default: self-hosted
         required: true
-        description: "Choose out of (self-hosted, ubuntu-20.04)"
+        description: "Where to run openzeppelin tests (self-hosted, ubuntu-20.04)"
```

Comment:
Technically, it's not about where to run, it's about what type of runner to use
Network combo box is more close to 'where'
btw, isn't the proposed label too long?